frontend_tests/casper_tests/11-admin.js

```diff
@@ -130,6 +130,52 @@ casper.waitWhileSelector('.emoji_row', function () {
     casper.test.assertDoesntExist('.emoji_row');
 });
 
+
+function get_suggestions(str) {
+    casper.then(function () {
+        this.echo("going to start");
+        casper.evaluate(function (str) {
+            $('.create_default_stream')
+            .focus()
+            .val(str)
+            .trigger($.Event('keyup', { which: 0 }));
+        }, str);
+    });
+}
+
+function select_from_suggestions(item) {
+    casper.then(function () {
+       casper.evaluate(function (item) {
+            var tah = $('.create_default_stream').data().typeahead;
+            tah.mouseenter({
+                currentTarget: $('.typeahead:visible li:contains("'+item+'")')[0]
+            });
+            tah.select();
+        }, {item: item});
+    });
+}
+
+//Test default stream creation and addition
+
+casper.then(function () {
+    casper.click('#settings-dropdown');
+    casper.click('a[href^="#subscriptions"]');
+    casper.click('#settings-dropdown');
+    casper.click('a[href^="#administration"]');
+    var stream_name = "Scotland";
+    get_suggestions("O");
```

Comment:
Ahh, OK, got it-- I misread it as a zero.  Can you just add a comment with that explanation of the reasoning on the line above this?